code/modules/clothing/glasses/hud.dm

```diff
@@ -55,6 +55,30 @@
 			user.say("WHY IS THERE A BAR ON MY HEAD?!!")
 	return OXYLOSS
 
+/obj/item/clothing/glasses/hud/civ
+	name = "civilian HUD"
+	desc = "A heads-up display that shows only the current job ID of the user."
+	icon_state = "civ"
+	hud_type = DATA_HUD_SECURITY_BASIC
```

Comment:
wouldnt you want to create your own hud type in the case the basic security huds get another thing?

Reply:
Basic security HUDs use the SECURITY_ADVANCED hud_type as well. That is the HUD layer that gives you all the security information. Nothing in the code currently uses SECURITY_BASIC, which is already designed to do exactly what I intend it to do. Adding a copy/paste of the code on the off-chance that this gets drastically changed in the future when it hasn't been touched in forever is bad spaghetti